packages/strapi-plugin-upload/admin/src/components/ListModal/index.js

```diff
@@ -33,12 +33,10 @@ const ListModal = () => {
     </BaselineAlignmentWrapper>
   );
 
+  const renderRightContent = noNavigation ? null : renderUploadModalButton;
```

Comment:
Same here, can't we put the noNavigation condition in the renderUploadModalButton function?